src/lib/VTXSPI/devVTXSPI.cpp

```diff
@@ -85,7 +85,22 @@ static void rtc6705WriteRegister(uint32_t regData)
     }
 
     #if defined(PLATFORM_ESP32)
-        vtxSPI->transferBits(regData, nullptr, 25);
+        // On some ESP32 MCUs there's a silicon bug which affects all 8n+1 bit and 1 bit transfers where the 
+        // last bit sent is corrupt.
+        // See: https://github.com/ExpressLRS/ExpressLRS/pull/2406#issuecomment-1722573356
+        //
+        // To reproduce, use an ESP32S3 and 25 bit transfers, change from channel A4 to A1, then A1 to A4 (ok), 
+        // then A4 to A3, then A3 to A4 (fail)
+        //
+        // 12816, 9286833 appears on the scope when changing from A:1->A:4
+        // 12816, 26064049 appears on the scope when changing from A:3->A:4
+        // 
+        // 9286833  = 0_1000_1101_1011_0100_1011_0001
+        // 26064049 = 1_1000_1101_1011_0100_1011_0001
+        //
+        // So, to workaround this issue, we send 32 bits, and the RTC67005 ignores the last 7 bits.
+        regData |= 0b1111111 << 25;
```

Comment:
Is this required when the bits are ignored?

Reply:
Meh, it's either all 1's on the LA with this, or leave it off and it's all 0's.

Reply:
> Is this required when the bits are ignored?

probably not, I didn't try, I added that line so that I could confirm that the ESP32S3 could actually send 32 bits of data on the scope.

would you like me to remove it?